run-vaxrank-b16-test-data.sh

```diff
@@ -4,4 +4,4 @@ vaxrank \
     --vaccine-peptide-length 25 \
     --mhc-predictor netmhc \
     --mhc-alleles H2-Kb,H2-Db \
-    --padding-around-mutation 5
+    --padding-around-mutation 5 --output-ascii-report vaccine-peptides-report.txt
```

Comment:
Definitely a nit, but maybe line-break this to keep it in line with the rest of the file?